### PR TITLE
FS-80: Adds errno to test failure output.

### PR DESCRIPTION
**Pull request by mufucaw**
Specifically for readwhatwaswritten.c, I need this to determine
what if any underlying issue is causing an mkdir failure within a
Valhalla PR.

### CODE REVIEW

File: tests/readwhatwaswritten.c [hunk 1]
```diff
@@ -252,12 +252,12 @@ int main(int argc, char *argv[]) {
 
     ret = mkdir(dirname, 0755);
     if (ret < 0) {
-        printf("FAIL: Couldn't make directory %s. Exiting\n", dirname);
+        printf("FAIL: Couldn't make directory %s (%i). Exiting\n", dirname, errno);
```
Comment (antonmashko) [inline tests/readwhatwaswritten.c:255]: are you sure about `%i`?
also I'd add string error message to print  
```
printf("FAIL: Couldn't make directory %s (%d) err=%s. Exiting\n", dirname, errno, strerror(errno));
```

Comment (mufucaw) [reply tests/readwhatwaswritten.c:255]: %i works for sure (it told me what the error actually was) but I think you're right about strerror(). I'll add it if we think it's worth merging this.

Comment (mufucaw) [reply tests/readwhatwaswritten.c]: Updated.

**Comment by djschaap** [comment]
I'd suggest closing or changing this to a draft PR - there isn't much interest in making any changes to fusedav at the moment.

**Comment by mufucaw** [comment]
This is really old. Closing.

**Pull request closed without merging**